myapp/utils/chains.py

```diff
@@ -0,0 +1,206 @@
+import operator
+import getpass
+import os
+
+from dotenv import load_dotenv
+from typing import Annotated, Sequence, TypedDict, List
+from pydantic import BaseModel, Field
+
+from langchain.output_parsers import PydanticOutputParser
+from langchain_core.prompts import PromptTemplate
+from langchain_core.pydantic_v1 import (
+    BaseModel as CoreBaseModel,
+    Field as CoreField,
+    validator,
+)
+from langchain_openai import ChatOpenAI
+from langchain_core.messages import BaseMessage
+
+from .util_var import (
+    phase_template,
+    feature_template,
+    qa_pair_template,
+    fix_array_template,
+    paraphrase_template,
+)
+
+
+from langchain_core.prompts import PromptTemplate
+from langchain_openai import ChatOpenAI
+from langchain_core.output_parsers import StrOutputParser
+
+load_dotenv()
+
+output_parser = StrOutputParser()
+
+prompt = PromptTemplate(
+    template=paraphrase_template,
+    input_variables=["requirement_document"],
+)
+
+model = ChatOpenAI(temperature=0, model="gpt-4o-mini")
+
+paraphrase_chain = prompt | model | output_parser
+
+
```

Comment:
this should be inside of class or a function, its not a good practice to to write scriptes like code inside django.